src/common/io/src/format_settings.rs

```diff
@@ -68,6 +68,8 @@ pub struct FormatSettings {
     pub json_quote_denormals: bool,
     pub json_escape_forward_slashes: bool,
     pub ident_case_sensitive: bool,
+
+    pub row_tag: Vec<u8>,
```

Comment:
`xml_row_tag` is better?

Reply:
agree, but we still using `row_tag` as a format option for the end user, since we are flowing snowflake style to not prefix format_name, https://docs.snowflake.com/en/sql-reference/sql/create-file-format.html